.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: scala
 scala:
   - "2.11.8"
+  - "2.12.1"
 jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
+script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc "project unicorn-core" +test
```

Comment:
no sure how to do it that only `unicorn-core` is cross-built; in current version it builds:
1) everything with 2.11.x once
2) unicorn-core with 2.11 and 2.12  (so 2.11 is built twice).

Kinda inelegant